### PR TITLE
feat(pci.data-processing): dashboard - add grafana button

### PR DESCRIPTION
**Pull request by jperchoc**
DATATR-13

Signed-off-by: Jonathan Perchoc <jonathan.perchoc@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/datatr-3-dataprocessing-jobs-dashboard f`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #DATATR-13
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Add a button to go to Grafana

## Related

```
DATATR-3
  ^ 
  | - DATATR-6
  | - DATATR-7
  | - DATATR-10
  | - DATATR-11
  | - DATATR-12
  | - DATATR-13
```
[DATATR-3](https://github.com/ovh/manager/tree/feat/datatr-3-dataprocessing-jobs-dashboard)
[DATATR-6](https://github.com/ovh/manager/pull/7120)
[DATATR-7](https://github.com/ovh/manager/pull/7115)
[DATATR-10](https://github.com/ovh/manager/pull/7116)
[DATATR-11](https://github.com/ovh/manager/pull/7119)
[DATATR-12](https://github.com/ovh/manager/pull/7117)
[DATATR-13](https://github.com/ovh/manager/pull/7118)

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/metrics-token/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "data_processing_details_metrics_token_modal_title": "Accéder au monitoring",
+  "data_processing_details_metrics_token_modal_grafana_button": "Accéder à Grafana",
   "data_processing_details_metrics_token_modal_dismiss_button": "Fermer",
   "data_processing_details_metrics_token_modal_description_1": "Vous pouvez accéder aux métriques détaillées de votre job via <a href=\"{{grafana_url}}\" target=\"_blank\">Grafana</a>.",
   "data_processing_details_metrics_token_modal_description_2": "<a href=\"{{docs_monitoring_url}}\">Voir la documentation</a>",
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/metrics-token/translations/Messages_fr_FR.json:6]: Maybe remove the html from the translation file.

**Comment by jperchoc** [comment]
> Hello @jperchoc can you please check Sonarqube feedback please?

Hi @JeremyDec, it should be ok now :wink:

**Review by Dimfacion — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/metrics-token/metrics-token.controller.js [hunk 2]
```diff
@@ -25,6 +26,10 @@ export default class {
     }
   }
 
+  goToGrafana() {
+    this.$window.open(this.grafanaUrl, '_blank');
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/metrics-token/metrics-token.controller.js:30]:
```suggestion
    this.$window.open(this.grafanaUrl, '_blank', 'noopener');
```

File: packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/metrics-token/metrics-token.html [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
     <span
         data-translate="data_processing_details_metrics_token_modal_description_1"
-        data-translate-values="{ grafana_url: $ctrl.grafanaUrl }"
-    >
-    </span>
-    <span
+    ></span>
+    <a href="{{$ctrl.grafanaUrl}}" target="_blank" rel="noopener">Grafana</a>.
+    <a
+        href="{{$ctrl.docsMonitoringUrl}}"
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/metrics-token/metrics-token.html:14]: No target `_blank` nor `rel="noopener"`?

File: packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/metrics-token/metrics-token.controller.js (hunk 2 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/metrics-token/metrics-token.controller.js]: Added here: https://github.com/ovh/manager/pull/7118/commits/186fb3f6c6c80e589be72c74e2a2f7ab5d435a76

File: packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/metrics-token/metrics-token.html (hunk 3 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/metrics-token/metrics-token.html:14]: Definitely missing, thanks ! Added here: https://github.com/ovh/manager/pull/7118/commits/186fb3f6c6c80e589be72c74e2a2f7ab5d435a76

**Review by anooparveti — APPROVED** [review]

**Pull request merged**